oio/xcute/jobs/mover.py

```diff
@@ -97,8 +97,8 @@ class RawxDecommissionJob(XcuteJob):
 
     @staticmethod
     def sanitize_params(params):
-        if not params.get('rawx_id'):
-            return ValueError('Missing rawx ID')
+        if not params.get('service_id'):
```

Comment:
I understand the change in the task, but why in the job as well? The job is still called `RawxDecommissionJob`, it should be treating `rawx_id`. Only the key in the payload should be changed.

Reply:
Rawx ID doesn't exist. The client only knows the service ID.